docs/team/zminghuiz.md

```diff
@@ -0,0 +1,27 @@
+# Zhu Minghui's Project Portfolio Page
+
+## Project: EconoCraft Pro
+
+EconoCraft is a single player text adventure game where a player takes on the role of a business owner or manager,
+tasked with growing their company from a small startup to a large corporation. The game would simulate real-world
+business challenges, including resource management, market analysis, product development, and financial planning to
+help students prepare for the future challenges.
+
+Given below are my contributions to the project.
+
```

Comment:
Nice format, but maybe you can add more htings since you have contributed so much to the  project

Reply:
Okay, thank you!